### ISSUE TITLE
Bottom-of-screen actions gone after last update (Pixel4XL)

### ISSUE THREAD

**Issue by sempervictus** [GrapheneOS/os-issue-tracker#559]
After the last update (stable channel), the bottom of my screen is misbehaving significantly - browsers all lost their navigation buttons, the task switcher doesn't work correctly anymore (screen flickers when trying to raise it), and there's a thin white bar, a couple of pixels tall and about a finger wide at the bottom of the screen which i'm guessing is an artifact of the underlying problem (its right where you'd pull up for the task switcher).
Have restarted the phone since the upgrade, no impact unfortunately.

**Comment by thestinger** [comment]
Please see https://grapheneos.org/releases#2021.05.16.04. Gesture mode is now the default navigation mode. There's an explanation of how it works there.

> enable gesture navigation by default (can change it via Settings → System → Gestures → System navigation) instead of legacy 2-button navigation: swipe left/right on the navigation bar to switch apps, swipe up for home screen, swipe up and hold for app overview (recent apps), swipe from the left/right edge to go back

2-button mode is a legacy mode and the stock OS now only has the older 3-button mode and gesture mode, with gesture mode as the default on the Pixel 4 and later. We decided it was time for us to move to it too, because it's a lot nicer once you get used to it. My experience is most people like gestures and a minority want the 3-button mode but most people don't like 2-button unless they got used to it on GrapheneOS since we're the only ones still using it.

**Comment by sempervictus** [comment]
Ah!  Thanks boss, i'm just spoiled i guess with how consistent the interface has been and didn't even occur to me that this might be a feature.

**Comment by thestinger** [comment]
We decided to finally align with the stock OS and move to gesture mode by default. It changed for you because you haven't ever set a mode yourself.

For Pixel 4 and later, stock OS only has 3-button mode and gesture mode as options. Gesture mode is the default there. 2-button mode was removed. AOSP still has 2-button mode but they took it away on newer devices and AOSP also doesn't enable it by default. We had to enable support for it and port over a 3-button mode improvement to it.
